content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -17,7 +17,10 @@ export class _DiscoveryStreamBase extends React.PureComponent {
       case "CardGrid":
         return (<CardGrid />);
       case "Hero":
-        return (<Hero />);
+        return (<Hero
+          feed={component.feed.url}
+          style={component.properties.style}
```

Comment:
Is this intended to be on every component as the way to override the local style from the server?  Or is it something else?

Reply:
I'm basing this off [this specification](https://docs.google.com/document/d/1RCVxPGY2EbOr3tG19MIpfAaA59ZOApt303XUI0-RMoM/edit#), which is what the [endpoint schema](https://app.swaggerhub.com/apis/PocketNewTab/FirefoxHome/0.2) is derived from. 

In this case it's just controlling whether or not the component has a border.

Reply:
PS: There's a lot more coming in this patch. Not ready for review yet, but questions are totally fine! :)